docs/open_source/release-process.md

```diff
@@ -195,22 +195,22 @@ We maintain a `next` branch on the MDC Web Catalog repository to keep ahead of b
 1. Push the chore branch and send a pull request for one last review
 1. Squash and merge the PR in GitHub
 1. Update your local `master` branch and deploy:
-  1. `git checkout master && git pull`
-  1. `npm start` if you want to double-check one last time (`master` should contain the same changes you tested in your PR)
-  1. `npm run deploy`
+   1. `git checkout master && git pull`
```

Comment:
this looks a lot better 👍

Reply:
It's what I intended, I just forgot that markdown detects nested lists by aligning the indentation with the text content of the outer list.